frontend/src/pages/forms/Form.jsx

```diff
@@ -30,8 +31,7 @@ const Form = ({ walletId, setWalletId }) => {
   const [loading, setLoading] = useState(false);
   const [alertMessage, setAlertMessage] = useState('');
   const [successful, setSuccessful] = useState(false);
-  useLockBodyScroll(successful);
-  const [showPopup, setShowPopup] = useState(false);
+  const [positionModal, setPositionModal] = useState(false);
 
   const { data: positionData, refetch: refetchPosition } = useQuery({
```

Comment:
this doesn't work, please test it before your next commit, axiosIntance should be used instead of axios, currently it doesn't do request to a backend..

Reply:
resolved